charmhelpers/contrib/openstack/audits/openstack_security_guide.py

```diff
@@ -83,16 +83,20 @@
     },
     'keystone': {
         # From security guide
-        '/etc/keystone/keystone.conf': {'group': 'keystone', 'mode': '640'},
+        '/etc/keystone/keystone.conf':
+            {'user': 'keystone', 'group': 'keystone', 'mode': '640'},
         '/etc/keystone/keystone-paste.ini':
-            {'group': 'keystone', 'mode': '640'},
-        '/etc/keystone/policy.json': {'group': 'keystone', 'mode': '640'},
-        '/etc/keystone/logging.conf': {'group': 'keystone', 'mode': '640'},
+            {'user': 'keystone', 'group': 'keystone', 'mode': '640'},
+        '/etc/keystone/policy.json':
+            {'user': 'keystone', 'group': 'keystone', 'mode': '640'},
+        '/etc/keystone/logging.conf':
+            {'user': 'keystone', 'group': 'keystone', 'mode': '640'},
         '/etc/keystone/ssl/certs/signing_cert.pem':
```

Comment:
In our implementation these certs will probably not be there, I assume non-existence of the file would make it pass?

Reply:
At the moment, actually, no. Missing file is going to fail - I'll modify this PR to allow for optional files to be included as well